app/assets/javascripts/govuk_publishing_components/lib/header-navigation.js

```diff
@@ -14,31 +20,42 @@
         e.preventDefault()
         var target = this.getAttribute('href') ? document.getElementById(this.getAttribute('href').substr(1)) : document.getElementById(this.getAttribute('data-search-toggle-for'))
         var targetClass = target.getAttribute('class') || ''
+        var isTargetSearchToggle = targetClass.match('search-toggle')
         var sourceClass = this.getAttribute('class') || ''
-        var isSearchToggle = sourceClass.match('search-toggle')
+        var isSourceSearchToggle = sourceClass.match('search-toggle')
         var showText = this.getAttribute('data-show-text') || 'Show search'
         var hideText = this.getAttribute('data-hide-text') || 'Hide search'
 
         if (targetClass.indexOf('js-visible') !== -1) {
           target.setAttribute('class', targetClass.replace(/(^|\s)js-visible(\s|$)/, ''))
-          if (window.GOVUK.analytics && window.GOVUK.analytics.trackEvent) {
-            window.GOVUK.analytics.trackEvent('headerClicked', 'menuClosed', { label: 'none' })
+          if (isTargetSearchToggle) {
+            sendClick('searchClosed')
+          } else {
+            sendClick('menuClosed')
           }
         } else {
           target.setAttribute('class', targetClass + ' js-visible')
-          if (window.GOVUK.analytics && window.GOVUK.analytics.trackEvent) {
-            window.GOVUK.analytics.trackEvent('headerClicked', 'menuOpened', { label: 'none' })
+          if (isTargetSearchToggle) {
+            sendClick('searchOpened')
+          } else {
+            sendClick('menuOpened')
           }
         }
         if (sourceClass.indexOf('js-visible') !== -1) {
           this.setAttribute('class', sourceClass.replace(/(^|\s)js-visible(\s|$)/, ''))
```

Comment:
On a header with both a menu and search, I'm seeing both `targetClass` and `sourceClass` being true when I click on the search toggle - and both false when clicking on the menu toggle. Going to dig a bit and see if I can find out why.